Misc/NEWS.d/next/Library/2018-10-25-09-59-00.bpo-35047.abbaa.rst

```diff
@@ -0,0 +1,2 @@
+``unittest.mock`` now prints the list of calls if ``assert_not_called``
```

Comment:
To me, "print" means writing into stdout. I would prefer to say something like "now includes mock calls in exception message".